source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -40,6 +40,14 @@ bool CLSBeamlineStatus::isOff() const
 	}
 }
 
+AMControl * CLSBeamlineStatus::beamlineStatusPVControl()
+{
+	if (beamlineStatusPVControl_ )
+		return beamlineStatusPVControl_ ;
+	else
+		return this;
```

Comment:
@dretrex I didn't agree with this changes. The purpose for this function is to return a signal control, which can be used to check the overall status of the beam. 

For that purpose, if there is a general PV control defined for beam status (`beamlineStatusPVControl_`), we should return that one; otherwise, the instance of `CLSBeamlineStatus` will behave for that purpose. The caller of `beamlineStatusPVControl()` should not be aware of whether they should use a general PV control or the CLSBeamlineStatus instance.

Btw, I will rename the function `beamlineStatusPVControl(()` to `beamlineStatusControl()`